pkgs/servers/jicofo/default.nix

```diff
@@ -1,11 +1,11 @@
-{ pkgs, lib, stdenv, fetchurl, dpkg, jre_headless, nixosTests }:
+{ lib, stdenv, fetchurl, dpkg, jre_headless, nixosTests }:
 
 let
   pname = "jicofo";
-  version = "1.0-612";
+  version = "1.0-692-hf-1";
```

Comment:
It looks like this version with the `-1` is at the end is inconsistent with what's going on with `jitsi-videobridge2`. Is the `-1` part of the version or not? Looking through https://download.jitsi.org/stable/ it looks like all releases of jicofo end with -1_all, so I think the way we had it before makes more sense.

Reply:
Changed it back